<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kennzeichengenerator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="gbOptions">
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pbLoadPreset">
           <property name="text">
            <string>Load Preset...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSavePreset">
           <property name="text">
            <string>Save Preset...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lText">
           <property name="text">
            <string>Text:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leText">
           <property name="toolTip">
            <string>The text that you want to show on the licence plate. Use &quot;:&quot; for the &quot;HU-Sticker&quot;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="cbNarrowFont">
           <property name="text">
            <string>Narrow Font</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lTextColor">
           <property name="text">
            <string>Text Color:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbTextColor">
           <item>
            <property name="text">
             <string>Black</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Red</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Green</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lType">
           <property name="text">
            <string>Type:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cbType">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>This isn't available now.</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lHuDate">
           <property name="text">
            <string>HU Date:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDateEdit" name="deHuDate">
           <property name="displayFormat">
            <string>MM.yyyy</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lStateEmblem">
           <property name="text">
            <string>State Emblem:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="cbStateEmblem">
           <item>
            <property name="text">
             <string>Empty</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Deutschland</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Baden-Würtemberg</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bayern</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Berlin</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Brandenburg</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bremen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hamburg</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hessen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mecklenburg-Vorpommern</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Niedersachsen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nordrhein-Westfalen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rheinland-Pfalz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Saarland</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sachsen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sachsen-Anhalt</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Schleswig-Holstein</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Thüringen</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lDisctrictName">
           <property name="text">
            <string>Disctrict Name:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="leDisctrictName"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="lImgHeight">
           <property name="text">
            <string>Resolution (Image height):</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="sbImgHeight">
           <property name="suffix">
            <string> px</string>
           </property>
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>4096</number>
           </property>
           <property name="value">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbOutput">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lPreview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pbSaveAsFile">
           <property name="text">
            <string>Save as File...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbCopyToClipboard">
           <property name="text">
            <string>Copy to Clipboard</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
